roles/georchestra/tasks/wars.yml

```diff
@@ -21,7 +21,41 @@
     update_cache: yes
     state: latest
   with_dict: "{{ georchestra_wars }}"
-  when: item.key != 'cadastrapp'
+  when: item.value.enabled|bool == True and item.key != 'cadastrapp' and item.value.pkg is defined
+
+- name: install the wars defined by an url
+  get_url:
+    url: "{{ item.value.url }}"
+    dest: "/usr/share/lib/georchestra-{{ item.key }}.zip"
+    headers:
+      Authorization: "Bearer {{ github_action_token }}"
+  with_dict: "{{ georchestra_wars }}"
+  when: item.value.enabled|bool == True and item.value.url is defined
+
+- name: unzips the war
+  unarchive:
+    src: "/usr/share/lib/georchestra-{{ item.key }}.zip"
+    dest: /usr/share/lib
+    remote_src: yes
+  with_dict: "{{ georchestra_wars }}"
+  when: item.value.enabled|bool == True and item.value.url is defined
+
+- name: creating symlinks for downloaded webapps
+  file:
+    owner: tomcat
+    src: "/usr/share/lib/{{ item.key }}{{ '-' if item.value.artifact_sha256 is defined }}{{ item.value.artifact_sha256 if item.value.artifact_sha256 is defined }}.war"
```

Comment:
that line blows for me on ansible 2.9.16, this works:
```
src: "/usr/share/lib/{{ item.key }}{% if item.value.artifact_sha256 is defined %}-{{ item.value.artifact_sha256  }}{% endif %}.war"
```

Reply:
applied locally already